human_info/src/config.py

```diff
@@ -10,13 +10,13 @@
 
 ERASE_LINE = '\x1b[2K'
 
-# possible FEATURES
-# ['nose', 'left eye', 'right eye', 'left ear', 'right ear', 'left shoulder', 'right shoulder', 'left elbow', 'right elbow', 'left wrist', 'right wrist', 'left hip', 'right hip', 'left knee', 'right knee', 'left ankle', 'right ankle']
+# possible FEATURE_DISTANCES
```

Comment:
It looks like a search and replace issue. This ones are just features. No distances
```suggestion
# possible FEATURE
```

Reply:
True